docs/src/main/sphinx/connector/kinesis.rst

```diff
@@ -95,6 +96,14 @@ The S3 bucket and folder will be checked every 10 minutes for updates and change
 
 This property is optional; the default is ``etc/kinesis``.
 
+``kinesis.table-description-refresh-interval``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+References the refresh interval foor an S3 URL or a folder within Trino deployment that holds one or more JSON files ending with ``.json``, which contain table description files.
+The S3 bucket and folder will be checked every 10 minutes for updates and changed files.
+
+This property is optional; the default is ``1 min``.
```

Comment:
This sentence is mostly duplicate of `kinesis.table-description-location` and it's little wrong. How about:
```
This property controls how often the table description is refreshed from S3.

This property is optional; the default is ``1m``.
```